ansible/default_vars.yml

```diff
@@ -9,8 +9,8 @@
 # Either true or false
 # https://www.nomadproject.io/docs/configuration/acl
 - name: Set nomad acl to "false" if it is not set by user
-  set_fact: nomad_agent_acl_enable="false"
-  when: nomad_agent_acl_enable is undefined
+  set_fact: nomad_acl="{{ lookup('env','TF_VAR_nomad_acl')|bool }}"
```

Comment:
How do you send this env variable from local machine into ansible provisioner? How we can use this for templates?

Reply:
If I understand your question correctly this is sent to the ansible provider by the fact that the makefile has `include .env` and `export`

Reply:
> If I understand your question correctly this is sent to the ansible provider by the fact that the makefile has `include .env` and `export`

In that case u run `vagrant up --provision` command with env variable the way 
like `TF_VAR_nomad_acl=true vagrant up --provision`, but this env does not transfered inside box as far as I know.

Reply:
yeah, `TF_VAR_nomad_acl=true vagrant up --provision` would be the same thing, and yes, ansible gets that variable. But you're correct, TF_VAR_nomad_acl would not be a system variable inside the box with this code

Reply:
> yeah, `TF_VAR_nomad_acl=true vagrant up --provision` would be the same thing, and yes, ansible gets that variable. But you're correct, TF_VAR_nomad_acl would not be a system variable inside the box with this code. I guess it would be nice with a systemwide `nomad_acl` variable as well, so that user-made terraform code could use the same setup as the main.tf in `template/test/terraform`

This could be done the same way it has been done so far with `VAULT_ADDR` etc. though,  so wouldn't be much of a problem to add